### PR TITLE
Remove last usages of async-stacktrace from PrairieLearn app

### PR DESCRIPTION
**Pull request by nwalters512**
This leaves just three files in `apps/grader-host` that use `async-stacktrace` 🎉 

I recommend reviewing this with whitespace changes hidden.

### CODE REVIEW

File: apps/prairielearn/src/tests/courseEditor.test.ts [hunk 1]
```diff
@@ -469,119 +468,39 @@ function testEdit(params) {
...
-      sqldb.queryOneRow(sql.select_last_job_sequence, [], (err, result) => {
-        if (ERR(err, callback)) return;
-        locals.job_sequence_id = result.rows[0].id;
-        callback(null);
-      });
-    });
-    it('should complete', function (callback) {
-      const checkComplete = function () {
-        const params = { job_sequence_id: locals.job_sequence_id };
-        sqldb.queryOneRow(sql.select_job_sequence, params, (err, result) => {
```
Comment (jonatanschroeder) [inline apps/prairielearn/src/tests/courseEditor.test.ts:571]: This query is no longer used in this file. Remove it from the .sql file?

File: apps/prairielearn/src/tests/fileEditor.test.ts [hunk 2]
```diff
@@ -869,71 +764,38 @@ function pullAndVerifyFileNotInDev(fileName) {
...
-      sqldb.queryOneRow(sql.select_last_job_sequence, [], (err, result) => {
-        if (ERR(err, callback)) return;
-        locals.job_sequence_id = result.rows[0].id;
-        callback(null);
-      });
-    });
-    it('should complete', function (callback) {
-      const checkComplete = function () {
-        const params = { job_sequence_id: locals.job_sequence_id };
-        sqldb.queryOneRow(sql.select_job_sequence, params, (err, result) => {
```
Comment (jonatanschroeder) [inline apps/prairielearn/src/tests/fileEditor.test.ts:923]: Again, query is no longer in use.

File: apps/prairielearn/src/tests/groupStudent.test.ts [hunk 3]
```diff
@@ -25,37 +24,33 @@ const storedConfig: Record<string, any> = {};
...
-        locals.assessmentUrl_2 = locals.courseInstanceUrl + '/assessment/' + locals.assessment_id_2;
-        locals.instructorAssessmentsUrlGroupTab_2 =
-          locals.courseInstanceUrl + '/instructor/assessment/' + locals.assessment_id_2 + '/groups';
-        callback(null);
-      });
+    it('should contain a group-based homework assessment', async () => {
+      const assessment_ids = await sqldb.queryRows(sql.select_group_work_assessment, IdSchema);
+      assert.lengthOf(assessment_ids, 2);
+      assert.notEqual(assessment_ids[0], undefined);
+      assert.notEqual(assessment_ids[1], undefined);
```
Comment (jonatanschroeder) [inline apps/prairielearn/src/tests/groupStudent.test.ts:43]:
```suggestion
      assert.isDefined(assessment_ids[0]);
      assert.isDefined(assessment_ids[1]);
```

**Review by jonatanschroeder — APPROVED** [review]

File: apps/prairielearn/src/tests/courseEditor.test.ts (hunk 1 above)
Comment (nwalters512) [reply apps/prairielearn/src/tests/courseEditor.test.ts:571]: Removed in e904ce9.

File: apps/prairielearn/src/tests/fileEditor.test.ts (hunk 2 above)
Comment (nwalters512) [reply apps/prairielearn/src/tests/fileEditor.test.ts:923]: Removed in e904ce9.

File: apps/prairielearn/src/tests/groupStudent.test.ts (hunk 3 above)
Comment (nwalters512) [reply apps/prairielearn/src/tests/groupStudent.test.ts]: Changed in e904ce9.

**Pull request merged**